scripts/runDbTests.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+set -eox pipefail
+
+source .env
+
+PGPASSWORD=$PGPASSWORD bash -c "pg_prove -d forumanalytics -h $DATABASE_HOST -U postgres --verbose $1"
```

Comment:
So this would need to be changed to work for LessWrong.

Reply:
Thinks for the catch